libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2505,6 +2505,8 @@ void IRGeneratorForStatements::appendExternalFunctionCall(
 			appendCode() << "mstore(add(" << m_utils.allocateUnboundedFunction() << "() , " << to_string(returnInfo.estimatedReturnSize) << "), 0)\n";
 	}
 
+	// NOTE: When the expected size of returndata is static, we pass that in to the call opcode an it gets copied automatically.
```

Comment:
```suggestion
	// NOTE: When the expected size of returndata is static, we pass that in to the call opcode and it gets copied automatically.
```
Looks like there was a typo in my suggestion. Sorry for that!